tests/bluetooth/bsim_bt/edtt_ble_test_app/gatt_test_app/prj.conf

```diff
@@ -26,3 +26,5 @@ CONFIG_BT_CTLR_RX_BUFFERS=3
 
 # To make DEVICE Name writable...
 CONFIG_BT_DEVICE_NAME_DYNAMIC=y
+
+CONFIG_BT_LL_SW_LLCP_LEGACY=y
```

Comment:
Why no GATT tests on new LLCP? i.e. no prj_dut_llcp.conf?

Reply:
The question is if GATT (and GAP) tests are effected by LLCP. For sake of completeness I can add a test config for the new LLCP

Reply:
see PR #40506 for running GATT and GAP tests with new LLCP